<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="FlagLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>541</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>573</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QLineEdit" name="LatitudeLE"/>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Population</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QComboBox" name="ltitudeCB">
      <item>
       <property name="text">
        <string>N</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>S</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="CapitalLE"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="HintB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Show hint</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="longitudeCB">
      <item>
       <property name="text">
        <string>W</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>E</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="HintCB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Country</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Capital</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Continent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Longitude</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Latitude</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Area</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Population</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Capital latitude</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="PopulationLE"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="CapitalLabel">
      <property name="text">
       <string>Capital</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="StartB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="HintLabel">
      <property name="text">
       <string>Hint</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="LongitudeLE"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Capital longitutde</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="CountryLabel">
      <property name="text">
       <string>Country</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ContinentLE"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="AreaLabel">
      <property name="text">
       <string>Area (kmÂ²)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ContinentLabel">
      <property name="text">
       <string>Continent</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="NextB">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="CountryLE"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="AreaLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>239</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="ScoreLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Score</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="correctAnsweLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
